### PR TITLE
Create allowing-multiple-school-accounts

### PR DESCRIPTION
**Pull request by claire-hughez**
A design history into how we want to allow for multiple school user accounts and no longer collect school induction tutor contact details in ECF2.

### CODE REVIEW

File: app/posts/ecf-v2/allowing-multiple-school-accounts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+In our discovery, access to the Manage training for early career teachers service was highlighted as a key problem for school users.
+
+In particular, being restricted to one account per school led to lots of support tickets and delays to getting ECTs and mentors registered.
+
+This is because:
+* there may be multiple people involved in the adminstration of early career teachers and mentors
+* the person that handles registration might change, and when it does, it can be hard to update once the singular user has left
+* the service asks for the school induction tutor to be the only user on the service, but often it is other administrative roles filling in this information
+
+Therefore, for the new digital service, we are going to allow multiple accounts per school. These will also be administered via DfE Sign In, [for reasons we detail in another design history.](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/exploring-using-dfe-sign-in/)
```
Comment (peteryates) [inline app/posts/ecf-v2/allowing-multiple-school-accounts:23]: Hmm, maybe:

```suggestion
In the new digital service we will allow multiple accounts per school. These will be administered by DfE Sign in, as discussed in the [Exploring using DfE Sign in](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/exploring-using-dfe-sign-in/) post.
```

File: app/posts/ecf-v2/allowing-multiple-school-accounts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Therefore we need to work out how we can surface contact details for one or more of the school users to lead providers via the API.
+
+This could be passing the emails for all school accounts, or just giving them the email of the user who registered that particular ECT or mentor.
+
+## Next steps
+
+We're still a long way off releasing anything. However, we still need to consider the impacts of this decision and how we can still best meet the needs of users in a way that is technichally feasible.
+
+First, we should check with further off stakeholders that there are negative ramifications of this decision we haven't considered. We're hoping this design history will help us to do this.
```
Comment (peteryates) [inline app/posts/ecf-v2/allowing-multiple-school-accounts:45]: Further off stakeholders?

**Review by peteryates — COMMENTED** [review]
Looks great. Couple of bits that might need a tweak.

File: app/posts/ecf-v2/allowing-multiple-school-accounts.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Therefore we need to work out how we can surface contact details for one or more of the school users to lead providers via the API.
+
+This could be passing the emails for all school accounts, or just giving them the email of the user who registered that particular ECT or mentor.
+
+## Next steps
+
+We're still a long way off releasing anything. However, we still need to consider the impacts of this decision and how we can still best meet the needs of users in a way that is technichally feasible.
+
+First, we should check with other teams in DfE that there are negative ramifications of this decision we haven't considered. We're hoping this design history will help us to do this.
```
Comment (claire-hughez) [inline app/posts/ecf-v2/allowing-multiple-school-accounts.md:45]:
```suggestion
First, we should check with other teams in DfE if there are negative ramifications of this decision we haven't considered. We're hoping this design history will help us to do this.
```

**Comment by gracecbrennan** [comment]
@JSJohal to review this DH.

**Review by JSJohal — APPROVED** [review]

**Pull request merged**